packages/create-bison-app/template/package.json.ejs

```diff
@@ -6,14 +6,15 @@
   "cacheDirectories": [".next/cache"],
 <% } -%>
   "scripts": {
-    "build": "ts-node ./scripts/buildProd",
+    "build": "node ./scripts/buildProd",
```

Comment:
Updated to a js file due to build errors that started occurring. The script did not appear to need to be typescript as it was already using exports.module syntax and an empty export.

Reply:
I fixed this in the CITJS repo by overriding the module definition in the CLI params: 

```"ts-node": "ts-node --compiler-options '{\"module\":\"CommonJS\"}'",```

Prisma has some notes on this:

https://www.prisma.io/docs/guides/application-lifecycle/seed-database#set-up-seeding-in-typescript